internal/proto/users/v1/users.proto

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) Abstract Machines
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+package users.v1;
+option go_package = "github.com/absmach/supermq/api/grpc/users/v1";
+
+// UsersService is a service that provides user-related functionalities
+// for SuperMQ services.
+service UsersService {
+  rpc SendEmailWithUserId(SendEmailWithUserIdReq) returns (SendEmailRes) {}
+}
+
+message SendEmailWithUserIdReq {
+  repeated string users = 1;    // Email recipients
+  string from = 2;              // Sender email address (optional)
+  string subject = 3;           // Email subject
+  string header = 4;            // Email header text
+  string user = 5;              // User name
+  string content = 6;           // Email content/URL
+  string footer = 7;            // Email footer text
+}
+
+message SendEmailRes {
+  bool sent = 1;                // Whether the email was sent successfully
```

Comment:
Let's use error here. So we have very go-ish interface where we inspect if response has an error.